test/navigation/navigators/registration/DeceasedSettlorRoutes.scala

```diff
@@ -97,14 +97,14 @@ trait DeceasedSettlorRoutes {
       }
     }
 
-    "go to AdditionalSettlorsYesNoPage from SettlorNationalInsuranceNumberPage" in {
+    "go to DeceasedSettlorAnswer from SettlorNationalInsuranceNumberPage" in {
       forAll(arbitrary[UserAnswers]) {
         userAnswers =>
 
           val answers = userAnswers.set(SetUpInAdditionToWillTrustYesNoPage, true).success.value
```

Comment:
@darrylmurphy Can you add a test where they're not answering the SetupInAdditionToAWillTrustYesNo as the proposed change requiring a Some(_) to the answer SetUpInAdditionToWillTrustYesNoPage will break the standard deceased journey. Adding a test will catch this in future.
These tests were originally without that answered, deceased route only. You may want to duplicate them and put them under "deceased in addition" or "deceased will trust"